### DISCUSSION TITLE
Enable LINK to RDP on device

### DISCUSSION THREAD

**Discussion by domenico-dig** [seriousm4x/UpSnap#662] [General]
It would be nice to have the ability to initiate RDP session to the device.
I try to use Link on device settings but not support protolol rdp://

**Comment by lexcorp** [comment]
Maybe this: pure GoLang RDP client

[(https://github.com/icodeface/grdp)](https://github.com/icodeface/grdp)

**Comment by lexcorp** [comment]
Maybe this:  Rust implementation of the Microsoft Remote Desktop Protocol (RDP)

[https://github.com/Devolutions/IronRDP](https://github.com/Devolutions/IronRDP)

**Comment by seriousm4x** [comment]
This will be part of the next release, see https://github.com/seriousm4x/UpSnap/commit/2d8fce32d39fd9bb0854b91bcb5b792c68f5cfdc
